src/messages/message_accumulator.rs

```diff
@@ -7,15 +7,15 @@
 // permissions and limitations relating to use of the SAFE Network Software.
 
 use crate::{
-    consensus::{AccumulationError, SignatureAccumulator},
     messages::{AccumulatingMessage, Message, PlainMessage, SrcAuthority},
     section::SectionProofChain,
 };
+use safe_network_signature_aggregator::{AccumulationError, SignatureAggregator};
```

Comment:
these have been re-exposed from the consensus mod.
no need to change here, or no need to re-expose inside consensus